core/services/ocr2/plugins/rebalancer/plugin.go

```diff
@@ -139,33 +161,14 @@ func (p *Plugin) Reports(seqNr uint64, outcome ocr3types.Outcome) ([]ocr3types.R
 	lggr := p.lggr.With("seqNr", seqNr)
 	lggr.Infow("in reports", "seqNr", seqNr)
 
-	obs, err := models.DecodeObservation(outcome)
+	outCome, err := models.DecodeOutcome(outcome)
 	if err != nil {
 		return nil, fmt.Errorf("decode outcome: %w", err)
 	}
 
-	lggr.Infow("computing transfers to reach balance",
-		"pendingTransfers", obs.PendingTransfers,
-		"liquidityGraph", p.liquidityGraph,
-		"liquidityPerChain", obs.LiquidityPerChain)
-
-	// compute a new graph with the median liquidities.
-	g, err := p.computeMedianGraph(obs.LiquidityPerChain)
-	if err != nil {
-		return nil, fmt.Errorf("compute median graph: %w", err)
-	}
-	if g.IsEmpty() {
-		return nil, fmt.Errorf("liquidity graph is empty, can't generate reports")
-	}
-
-	transfersToReachBalance, err := p.liquidityRebalancer.ComputeTransfersToBalance(g, obs.PendingTransfers)
-	if err != nil {
-		return nil, fmt.Errorf("compute transfers to reach balance: %w", err)
-	}
-
 	// group transfers by source chain
 	transfersBySourceNet := make(map[models.NetworkSelector][]models.Transfer)
-	for _, tr := range transfersToReachBalance {
+	for _, tr := range outCome.TransfersToReachBalance {
```

Comment:
No need to capitalize the `C` in `outcome`
```suggestion
	for _, tr := range outcome.TransfersToReachBalance {
```